webapp/portlet/src/main/webapp/activity-reactions/components/ActivityReactionsListItems.vue

```diff
@@ -89,6 +95,18 @@ export default {
           console.error('Error processing action', e);
         });
     },
+    initTiptip() {
+      this.$nextTick(() => {
+        $(`#${this.cmpId}`).userPopup({
+          restURL: '/portal/rest/social/people/getPeopleInfo/{0}.json',
```

Comment:
Please check if you do not need more initiation attributes ? labels for example

Reply:
Okay